pkg/reconciler/v1alpha1/autoscaling/kpa_scaler.go

```diff
@@ -192,3 +181,19 @@ func (rs *kpaScaler) Scale(ctx context.Context, kpa *kpa.PodAutoscaler, desiredS
 	logger.Debug("Successfully scaled.")
 	return nil
 }
+
+func (ks *kpaScaler) updateServingState(logger *zap.SugaredLogger, namespace, name string, state v1alpha1.RevisionServingStateType) error {
```

Comment:
We should be able to drop this entirely and deprecate `ServingState`.  We can't remove it from Revision until we have CRD versioning and push for `v1beta1`